project/cmake/modules/FindMDNSEmbedded.cmake

```diff
@@ -0,0 +1,38 @@
+#.rst:
+# FindMDNSEmbedded
+# ----------------
+# Finds the mDNSEmbedded library
+#
+# This will will define the following variables::
+#
+# MDNSEMBEDDED_FOUND - system has mDNSEmbedded
+# MDNSEMBEDDED_INCLUDE_DIRS - the mDNSEmbedded include directory
+# MDNSEMBEDDED_LIBRARIES - the mDNSEmbedded libraries
+# MDNSEMBEDDED_DEFINITIONS - the mDNSEmbedded definitions
+#
+# and the following imported targets::
+#
+#   MDNSEmbedded::MDNSEmbedded   - The mDNSEmbedded library
+
+find_path(MDNSEMBEDDED_INCLUDE_DIR NAMES mDnsEmbedded.h)
+find_library(MDNSEMBEDDED_LIBRARY NAMES mDNSEmbedded)
+
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(mDNSEmbedded
+                                  REQUIRED_VARS MDNSEMBEDDED_LIBRARY MDNSEMBEDDED_INCLUDE_DIR)
+
+if(MDNSEMBEDDED_FOUND)
+  set(MDNSEMBEDDED_INCLUDE_DIRS ${MDNSEMBEDDED_INCLUDE_DIR})
+  set(MDNSEMBEDDED_LIBRARIES ${MDNSEMBEDDED_LIBRARY})
+  set(MDNSEMBEDDED_DEFINITIONS -DHAVE_LIBMDNSEMBEDDED=1)
```

Comment:
From https://cmake.org/cmake/help/latest/manual/cmake-developer.7.html#find-modules
> Xxx_DEFINITIONS
    Definitions to use when compiling code that uses Xxx. This really shouldn’t include options such as -DHAS_JPEG that a client source-code file uses to decide whether to #include <jpeg.h>

Is it really needed?
Otherwise looks good.